akka-stream/src/main/scala/akka/stream/impl/io/FileSubscriber.scala

```diff
@@ -40,7 +37,6 @@ private[akka] class FileSubscriber(f: File, completionPromise: Promise[IOResult]
   private var bytesWritten: Long = 0
 
   override def preStart(): Unit = try {
-    val openOptions = if (append) FileSubscriber.Append else FileSubscriber.Write
     chan = FileChannel.open(f.toPath, openOptions)
```

Comment:
I'd vote for `import scala.collection.JavaConverters._` + `asJava` here instead of the `JavaConversions` import above.